week-1/implement/dedupe.js

```diff
@@ -1 +1,14 @@
-function dedupe() {}
+function dedupe(array) {
+    
+    const uniqueElements = new Set(array);
+  
+    
+    const dedupedArray = [...uniqueElements];
+  
+    return dedupedArray;
+  }
```

Comment:
A nice and clean implementation for the solution. Well done Laura. The only thing I want is:

Can you please tell me what is the reason you have chosen to use a `Set` in the line `const uniqueElements = new Set(array);` instead of an array function? There is nothing wrong about using it, I just wanted to know your thought process behind it.